### DISCUSSION TITLE
How did you get into the arch linux community repo?

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ytfzf#655] [Q&A] [answered]
Was wondering how that happened.
Do you need connections to an insider?

Greetings from ani-cli

**Comment by Euro20179** [comment, marked as answer]
Honestly Im not sure, I think the right person just happened to see it.

also,
you guys to great web scraping work on the ani-cli project, it's my goto for anime.

**Reply by port19x** [reply]
thx :smile:
